code/script.js

```diff
@@ -1,51 +1,312 @@
+  // the if statement checks if the sender is 'user' and if that's the case it inserts an html section inside the chat with the posted message
   if (sender === 'user') {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
-          <p>${message}</p>
+          <p class="p-bubble-msg">${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/bot-guest.svg" alt="User" />  
       </section>
     `
-    // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
+    // the else if statement checks if the sender is a bot and if that's the case it inserts an html section inside the chat with the posted message
   } else if (sender === 'bot') {
+    //console.log('bot-message') //added console log
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
-        <div class="bubble bot-bubble">
-          <p>${message}</p>
+        <img src="assets/bot-hotel.svg" alt="Bot" />
+            <div class="bubble bot-bubble">
+              <p class="p-bubble-msg">${message}</p>
         </div>
       </section>
     `
-  }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+  } 
+    // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box and adds sound
   chat.scrollTop = chat.scrollHeight;
+  messageSound();
 }
 
-// Starts here
+
+// Conversations starts here
+
+// Question 1 - Bot asks 'What is your room number'?
+
 const greeting = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+      showMessage("Welcome to Hotel Vosje. What is your room number?", 'bot'),2000; 
+    };
+  
+
+// User answers with room number
+
+const handleFormInput = (event) => {        // at submit this function will be invoked
+  event.preventDefault();        // prevents website refresh at submit
+  const room = roomInput.value;        // input value will be stored in the const name
+  showMessage(`My room number is ${room}`, 'user');      // users answer 
+  roomInput.value = '';        // clearing room input setting it to an empty string
+
+  if (room === '') {       // this will prompt the user to answer correctly
+    showMessage(`Please, give me your room number.`, 'bot');
+  } else {
+    greetingLoop(room);
+  }
+}
+
+// The bot answers hello room #. This const will also be used as loop back selections if guest needs something else
+
+const greetingLoop = (room) => {
+  typing(); 
```

Comment:
What means this "typing"? Is it to make look like a bot it's thinking and typing?

Reply:
yes the typing() is to invoke the function that was created before that to make it look like the bot is thinking and typing here :)